### PR TITLE
fix: disable stake and unstake

### LINKED ISSUE #1322: stake/unstake swapr token button doesnt work
**Description**
stake/unstake swapr token button doesnt work

#### Definition of done
- [ ] Description is created (Task creator)
- [ ] Code is complete (Developer)
- [ ] Scope of test and what was done section is defined in task (Developer)
- [ ] Review passed (Code reviewer)
- [ ] Verified manually (QA)
- [ ] Test cases created/updated if necessary (QA)
- [ ] Change successfully merged
- [ ] Release notes added if needed

**Recording**

https://images.zenhubusercontent.com/209400598/8b8743ee-7847-4007-98a4-e6b7b40f0728/s.mp4

### PR DESCRIPTION
**Pull request by berteotti**
# Summary

Fixes #1322

Disable stake and unstake buttons on swapr details modal

image

### CODE REVIEW

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -178,6 +183,10 @@ export const ButtonPurple = styled(Base)`
   &:active {
     background-color: ${({ theme }) => darken(0.1, theme.dark2)};
   }
+  &:disabled {
```
Comment (Diogomartf) [inline src/components/Button/Button.tsx:186]: Maybe we can add &:disabled styles to `Base`

```
 &:disabled {
    opacity: 50%;
    cursor: not-allowed;
  }
```

**Comment by MilanVojnovic95** [comment]
Tested and looks ok to me 👍

File: src/components/Button/Button.tsx (hunk 1 above)
Comment (berteotti) [reply src/components/Button/Button.tsx:186]: The things is, there are so many different behaviours to disabled Buttons, I don't know if I can break something if I move it to `Base`. Maybe we could have an issue to clean this Button file and have better components. Will see what I can do.

**Comment by niemam29** [comment]
Looks good to me

**Review by Diogomartf — APPROVED** [review]

**Review by wixzi — APPROVED** [review]

**Pull request merged**